### PR TITLE
add vultr

### PR DESCRIPTION
**Pull request by shkadov**

### CODE REVIEW

File: vultr/terraform/cluster.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    node_quantity = 1
+    plan          = var.node_pools_plan
+    label         = var.label
+    auto_scaler   = var.auto_scaler
+    min_nodes     = var.min_node
+    max_nodes     = var.max_node
+  }
+}
+
+resource "vultr_kubernetes_node_pools" "this" {
```
Comment (bashofmann) [inline vultr/terraform/cluster.tf]: This node pool is duplicated. 

I get his error:

```
│ Error: error creating node pool: gave up after 4 attempts, last error: "{\"error\":\"VKE server error: Cannot create NodePool: Cannot add NodePool: cannot have duplicate node pool labels\",\"status\":500}"
```

Removing this works then.

File: vultr/terraform/cluster.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
+resource "vultr_kubernetes" "this" {
```
Comment (bashofmann) [inline vultr/terraform/cluster.tf]: Also output the kubeconfig like this

```
resource "local_file" "kubeconfig" {
  content  = base64decode(vultr_kubernetes.this.kube_config)
  filename = "kubeconfig"
}
```

File: vultr/terraform/variables.tf [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+variable "label" {
+  description = "The VKE cluster's label."
+  type        = string
+  default     = "qdrant-example"
+}
+
+variable "k8s_version" {
+  description = "The version of your VKE cluster you want deployed"
+  type        = string
+  default     = "v1.29.1+1"
```
Comment (bashofmann) [inline vultr/terraform/variables.tf]: This is already outdated

```suggestion
  default     = "v1.29.2+1"
```

File: vultr/terraform/terraform.tfvars.example [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+api_key = ""
+
+# The region your VKE cluster will be deployed in
+region = "ewr"
+
+# The VKE cluster's label.
+label = "qdrant-example"
+
+# The version of your VKE cluster you want deployed
+k8s_version = "v1.29.1+1"
```
Comment (bashofmann) [inline vultr/terraform/terraform.tfvars.example]:
```suggestion
k8s_version = "v1.29.2+1"
```

File: vultr/terraform/cluster.tf (hunk 2 above)
Comment (bashofmann) [inline vultr/terraform/cluster.tf]: Please pick a better name for the resource instead of `this`.

File: vultr/terraform/vpc.tf [hunk 5]
```diff
@@ -0,0 +1,6 @@
+resource "vultr_vpc" "this" {
```
Comment (bashofmann) [inline vultr/terraform/vpc.tf]: Please pick a better name for the resource instead of `this`.

File: vultr/terraform/cluster.tf (hunk 1 above)
Comment (shkadov) [reply vultr/terraform/cluster.tf]: done

File: vultr/terraform/cluster.tf (hunk 2 above)
Comment (shkadov) [reply vultr/terraform/cluster.tf]: done

Comment (shkadov) [reply vultr/terraform/cluster.tf]: re-named

File: vultr/terraform/vpc.tf (hunk 5 above)
Comment (shkadov) [reply vultr/terraform/vpc.tf]: re-named

**Review by bashofmann — APPROVED** [review]

**Pull request merged**